src/data/__tests__/calculate.test.ts

```diff
@@ -135,36 +135,38 @@ describe('calculate', () => {
     ${'Indoor unmasked hangout with 2 other people'}                  | ${720 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Car ride with 1 other person for 15 mins'}                     | ${90 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'One-night stand with a random person'}                         | ${2880}
-    ${'Live-in partner who has no indoor interactions besides you'}   | ${21.6 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Grocery store for 60 minutes (average # of shoppers)'}         | ${90 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Grocery store for 60 minutes (few other shoppers)'}            | ${45 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Plane ride (full flight)'}                                     | ${369 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Plane ride (middle seat empty)'}                               | ${180 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Live-in partner who has no indoor interactions besides you'}   | ${29 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Grocery store for 60 minutes (average # of shoppers)'}         | ${80 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Grocery store for 60 minutes (few other shoppers)'}            | ${40 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Plane ride (full flight)'}                                     | ${328 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Plane ride (middle seat empty)'}                               | ${160 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Eating in restaurant, outdoors'}                               | ${202.5 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Eating in restaurant, indoors'}                                | ${4050 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Going to bar'}                                                 | ${27000 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Outdoor party: 80 people, masked, with 3 feet between people'} | ${1080 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Outdoor party: 80 people, masked, with 3 feet between people'} | ${960 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Indoor party: 25 people, unmasked'}                            | ${27000 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Outdoor, masked hangout with person who has COVID'}            | ${750 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Outdoor, masked hangout with person who has COVID'}            | ${666.6 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Indoor, unmasked hangout with person who has COVID'}           | ${60000 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Voting in-person'}                                             | ${3 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Voting in-person'}                                             | ${2.6 * B117_CONTAGIOUSNESS_ADJUSTMENT}
   `('should return $result for $scenario', ({ scenario, result }) => {
     const data: CalculatorData = testData(prepopulated[scenario])
 
-    expect(calcValue(data)).toBeCloseTo(result)
+    expect(calcValue(data)).toBeCloseTo(result, 0)
```

Comment:
@beshaya I set this to zero decimal places. now that we have 1/3 and 2/3 in the multipliers, the decimals kept getting all weird. I was having to put in stuff like `327.997 * B117_CONTAGIOUSNESS_ADJUSTMENT` -- So I just set it to zero. Let me know if that presents any concerns for you. You have your head around the test suite better than I do.

Reply:
ah, that's fine.